yt/frontends/amrvac/tests/test_output.py

```diff
@@ -0,0 +1,39 @@
+import numpy as np
+
+import yt
+from yt.utilities.parameter_file_storage import output_type_registry
+from yt.utilities.answer_testing.framework import requires_ds, data_dir_load
+
+blastwave_cartesian_3D = "amrvac/bw_3d0000.dat"
+blastwave_polar_2D = "amrvac/bw_polar_2D0000.dat"
+blastwave_spherical_2D = "amrvac/bw_2d0000.dat"
+blastwave_cylindrical_3D = "amrvac/bw_cylindrical_3D0000.dat"
+khi_cartesian_2D = "amrvac/kh_2D0000.dat"
+khi_cartesian_3D = "amrvac/kh_3D0000.dat"
+jet_cylindrical_25D = "amrvac/Jet0003.dat"
+riemann_cartesian_175D = "amrvac/R_1d0005.dat"
```

Comment:
Since your tests only require the `blastwave_cartesian_3D` dataset, I think you don't need these other definitions in the file. If you add tests for these other datasets, you can add the definitions as needed.

Reply:
sure. I'll either clean this up or figure out how to move our tests there. I don't think I ever used generators before so there's definitely going to be a little challenge for me here but I'm on it.